templates/android/template/app/src/main/java/org/haxe/lime/GameActivity.java

```diff
@@ -137,6 +139,14 @@ protected void onCreate (Bundle state) {
 
 	@Override protected void onDestroy () {
 
+		if (vibrator != null) {
+			
+			Log.d ("GameActivity", "Cancelling vibration");
+
+			vibrator.cancel ();
+
+		}
+
```

Comment:
Is this necessary? [The documentation](https://developer.android.com/reference/android/os/Vibrator) claims ongoing vibrations will be stopped automatically.

Reply:
> Is this necessary? [The documentation](https://developer.android.com/reference/android/os/Vibrator) claims ongoing vibrations will be stopped automatically.

I'll remove it then